azure-pipelines.yml

```
@@ -0,0 +1,37 @@
+# ASP.NET Core (.NET Framework)
+# Build and test ASP.NET Core projects targeting the full .NET Framework.
+# Add steps that publish symbols, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/dotnet-core
+
+trigger:
+- Assignment1
```

You may want to consider matching on more than just this single branch name so you can leverage this for future assignments. 
```suggestion
- Assignment*
```